book/tipps-und-tricks.md

```diff
@@ -6,5 +6,55 @@ Hier kannst du zu konkreten Problemstellungen  nachlesen, wie du sie lösen kann
 Zufallszahlen
 -------------
 
+Für Spiele und Rätsel mit dem Tiptoi-Stift ist es häufig nötig, Aktionen zufallsgesteuert auszuführen. Auch wenn bisher die Fähigkeiten des Stifts in dieser Hinsicht noch nicht ganz verstanden sind, gibt es ein paar Techniken, die man hier einsetzen kann.
+
+### Timer
+
+Der Stift verfügt über eine Art Timer, den man per Befehl `T()` abfragen kann:
+
+    # Timer-Wert in $register speichern
+    T($register, 65535)
+
+**Syntax:** T(_register_, _modulo_)
```

Comment:
Im finalen Werk wird es eine Referenz mit allen Befehlen geben (vorheriges Kapitel), so dass man an der Stelle vielleicht besser darauf verweist und den Leser nur kurz erinnert, was der Befehl macht. Aber das können wir Ändern wenn die Referenz fertig ist.

Reply:
Ich kann natürlich auch gerne diese Beschreibung schon in das Referenz-Kapitel übertragen und dann darauf verweisen. Soll ich?

Reply:
Ne, lass erstmal so. Ich will mir erst Gedanken machen wie das Referenz-Kapitel aufgebaut wird.